content/spin/running-apps.md

```diff
@@ -74,6 +75,45 @@ $ spin up --log-dir ~/dev/bugbash
 
 Some trigger types support additional `spin up` flags.  For example, HTTP applications can have a `--listen` flag to specify an address and port to listen on.  See the [HTTP trigger](http-trigger) and [Redis trigger](redis-trigger) pages for more details.
 
+## The Spin Watch Feature
```

Comment:
Suggest "The `watch` Command" or "Monitoring Applications for Changes"

Reply:
Thanks, went with "Monitoring Applications for Changes"